behavior.py

```diff
@@ -0,0 +1,103 @@
+import datajoint as dj
+
+import actions
+import reference
+import equipment
+
+
+schema = dj.schema(dj.config['names.%s' % __name__], locals())
+
+'''
+TODO: TimeScale items missing wrt TimeSeries items, which were:
+- PupilTracking.xyd
+- PupilTracking.movie
+- HeadTracking.xy_theta
+- HeadTracking.movie
+TimeScale not yet defined
+'''
+
+
+@schema
+class Movie(dj.Manual):
```

Comment:
need to double check if these are stimulus videos or observation videos..
from memory it seems they were stimulus, so I broke out to prevent storing the same large blob in each record, esp. since the video could be reused among different experimental sessions

Reply:
In case these are stimulus videos, I agree that it'd make sense to break it out into its own table. However, given the structure, the movie field of the pupil tracking is most likely referring to the vide of the pupil and hence it would make the most sense for the movie to be directly a blob (or more likely an external) inside the PupilTracking table.

Reply:
Indeed this seems to be the the case -
I think was bogged down w/r/t 'TimeSeries', which in turn is a 'DataSet', and so saw the link
as a potentially 1:M relationship with respect to movies. updating accordingly.
```
    movie = models.ForeignKey(TimeSeries, blank=True, null=True,
                              related_name="pupil_tracking_movie",
                              help_text="Link to raw data")
```